.github/workflows/tests.yaml

```diff
@@ -0,0 +1,330 @@
+  #           docker-credential-gcr configure-docker
+  #     - name: Tag the Docker image and push it to Google Container Registry.
+  #       run: |
+  #           if [ -n "$CIRCLE_TAG" ]; then
+  #             export MAJOR=`echo $CIRCLE_TAG | awk -F '.' '{ print $1; }'`
+  #             export MINOR=`echo $CIRCLE_TAG | awk -F '.' '{ print $2; }'`
+  #             export PATCH=`echo $CIRCLE_TAG | awk -F '.' '{ print $3; }'`
+  #             docker tag gcr.io/gapic-images/gapic-generator-python:latest gcr.io/gapic-images/gapic-generator-python:$MAJOR.$MINOR.$PATCH
+  #             docker tag gcr.io/gapic-images/gapic-generator-python:latest gcr.io/gapic-images/gapic-generator-python:$MAJOR.$MINOR
+  #             docker tag gcr.io/gapic-images/gapic-generator-python:latest gcr.io/gapic-images/gapic-generator-python:$MAJOR
+  #             docker push gcr.io/gapic-images/gapic-generator-python:$MAJOR.$MINOR.$PATCH
+  #             docker push gcr.io/gapic-images/gapic-generator-python:$MAJOR.$MINOR
+  #             docker push gcr.io/gapic-images/gapic-generator-python:$MAJOR
+  #           fi
+  #           docker push gcr.io/gapic-images/gapic-generator-python:latest
+  showcase:
+    strategy:
+      matrix:
+        target: [showcase, showcase_alternative_templates]
+    # needs:
+    #   - [docs, mypy, unit, showcase-unit, showcase-mypy]
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+      - name: Install system dependencies.
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y curl pandoc unzip gcc
+      - name: Install showcase
+        run: |
+          sudo mkdir -p /usr/src/showcase
+          sudo chown -R ${USER} /usr/src/
+          curl --location https://github.com/googleapis/gapic-showcase/releases/download/v0.12.0/gapic-showcase-0.12.0-linux-amd64.tar.gz --output /usr/src/showcase/showcase-0.12.0-linux-amd64.tar.gz
+          cd /usr/src/showcase/
+          tar -xf showcase-*
+          ./gapic-showcase run &
+          cd -
+      - name: Install nox.
+        run: python -m pip install nox
+      - name: Install protoc 3.12.1.
+        run: |
+            sudo mkdir -p /usr/src/protoc/
+            sudo chown -R ${USER} /usr/src/
+            curl --location https://github.com/google/protobuf/releases/download/v3.12.1/protoc-3.12.1-linux-x86_64.zip --output /usr/src/protoc/protoc-3.12.1.zip
+            cd /usr/src/protoc/
+            unzip protoc-3.12.1.zip
+            sudo ln -s /usr/src/protoc/bin/protoc /usr/local/bin/protoc
+      - name: Run showcase tests.
+        run: nox -s ${{ matrix.target }}
+  showcase-mtls:
+    strategy:
+      matrix:
+        target: [showcase_mtls, showcase_mtls_alternative_templates]
+    # needs:
+    #   - [docs, mypy, unit, showcase-unit, showcase-mypy]
```

Comment:
Should this be deleted or is it a palceholder?

Reply:
It's just cruft. Removing.